mobile/src/screens/auth/components/label/label.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Text, View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+const Label: React.FC = () => {
+  return (
+    <View style={styles.labelContainer}>
```

Comment:
move all labelContainer styles to labelText styles and label.tsx should look like this after changes:
import React from 'react';

import { Text } from '#libs/components/components';

import { styles } from './styles';

const Label: React.FC = () => {
  return <Text style={styles.labelText}>CalmPal</Text>;
};

export { Label };